prov/sockets/src/sock_fabric.c

```diff
@@ -735,11 +735,13 @@ static int sock_addr_matches_interface(struct slist *addr_list,
 
 static int sock_node_matches_interface(struct slist *addr_list, const char *node)
 {
-	union ofi_sock_ip addr = { 0 };
+	union ofi_sock_ip addr;
 	struct addrinfo *rai = NULL, ai = {
 		.ai_socktype = SOCK_STREAM,
 	};
 
+	memset(&addr, 0, sizeof(addr));
+
```

Comment:
I just updated my PR to address this and the windows build issues.  I'm testing the windows build via travis now.